Igor_Shahov/14.K8s.Jenk_Argo_WS/jenkins-app/values.yaml

```diff
@@ -0,0 +1,26 @@
+namespace: ci-cd
+name: jenkins
+image: jfrog.it-academy.by/public/jenkins-ci:shahov
+
+
+nfs_path: /mnt/IT-Academy/nfs-data/sa2-21-22/shahov/jenkins
+nfs_server: 192.168.37.105
+
+
+ingress:
+  enabled: enabled
+ing_alias: "jenkins-helm.k8s-32.sa"
+ing_host: jenkins-helm.k8s-31.sa
+ing_port: 8080
+
+resources:
```

Comment:
please use nested yaml object for resources: 
https://github.com/pluhin/sa.it-academy.by/blob/md-sa2-21-22/demo/13.K8s/first-app/values.yaml#L60-L66
and in template:
https://github.com/pluhin/sa.it-academy.by/blob/md-sa2-21-22/demo/13.K8s/first-app/templates/deployment.yaml#L49